src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs

```diff
@@ -42,9 +42,27 @@ public static IResourceBuilder<RedisResource> AddRedis(this IDistributedApplicat
 
         var redis = new RedisResource(name);
 
+        return builder.AddResource(redis).ApplyRedis(port);
+    }
+
+    /// <summary>
+    /// Configures a container resource for Redis container.
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <param name="port"></param>
+    /// <param name="configureContainer"></param>
+    /// <returns></returns>
+    /// <exception cref="DistributedApplicationException"></exception>
+    /// <exception cref="InvalidOperationException"></exception>
+    public static IResourceBuilder<RedisResource> ApplyRedis(this IResourceBuilder<RedisResource> builder, int? port = null, Action<IResourceBuilder<RedisResource>>? configureContainer = null)
```

Comment:
@eerhardt I don't think this will be the final API shape but the idea here is to get the redis defaults applied to an existing builder.

Reply:
We don't expect "normal" customers to call this method right? Maybe it shouldn't be an extension method so it doesn't clutter up the Intellisense for normal AppHost developers.